src/components/NavigationTop/NavigationTop.jsx

```diff
@@ -33,6 +35,16 @@ const NavigationTop = () => {
     localStorage.setItem("halcyon-gaming-theme", theme);
   }, [theme]);
 
+  useEffect(() => {
+    if(hamburgerOpen) {
+      setHamburgerOpen(false);
+    }
+  }, [location]);
```

Comment:
Instead of an if check, you can simply do `setHamburgerOpen(false);` in the useEffect().

Reply:
done